starter-code/functions-and-arrays.js

```diff
@@ -20,21 +20,45 @@ var words = [
   "orchard",
   "crackpot"
 ];
+
+
+function findLongestWord (words) {
+  var j = "";
+    for(var i=0; i < words.length; i++){
+        if( words[i].length > j.length) {
+            j = words[i];
+          }
+    }
+
+  return j
```

Comment:
try to give variables a name that makes sense, in this case, something like "longestWord"